sdk/client/commander.go

```diff
@@ -314,27 +315,28 @@ func (c *commander) createClient() error {
 func (c *commander) recvLoop() {
 	log.Debug("Commander receive loop starting")
 	for {
-		err := sdk.WaitUntil(c.ctx, c.backoffSettings.initialInterval, c.backoffSettings.maxInterval, c.backoffSettings.maxTimeout, func() error {
-			cmd, err := c.channel.Recv()
-			log.Infof("Commander received %v, %v", cmd, err)
-			if err != nil {
-				return c.handleGrpcError("Commander Channel Recv", err)
-			}
+		err := sdk.WaitUntilWithJitterAndMultiplier(c.ctx, c.backoffSettings.initialInterval, c.backoffSettings.maxInterval,
+			c.backoffSettings.maxTimeout, c.backoffSettings.randomization_factor, c.backoffSettings.multiplier, func() error {
+				cmd, err := c.channel.Recv()
+				log.Infof("Commander received %v, %v", cmd, err)
+				if err != nil {
```

Comment:
if err != nil then wouldn't cmd always be nil since there is an error getting messages from the channel?

Reply:
Command server and metrics server from our end would send gRPC Unavailable status error code if server is not accepting request for certain duration. I can provide link to the code separately.

And at the same time, we want to provide PAUSE command. 

So in situation like this, how do you think we can capture PAUSE command.